### ISSUE TITLE
Does ng2-table work in Angular4

### ISSUE THREAD

**Issue by OhioGuy67** [valor-software/ng2-table#583]
Is there any reason that ng2-table won't work in Angular4?

**Comment by narsivc** [comment]
bump. Has anyone tried this with Angular 4 ?

**Comment by jquince** [comment]
Yes just trying it with  4.3.6 and some demo code.   Seems to work fine so far.

**Comment by nblavoie** [comment]
It does, without any issues others than these :

`npm WARN ng2-table@1.3.2 requires a peer of @angular/common@~2.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-table@1.3.2 requires a peer of @angular/compiler@~2.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-table@1.3.2 requires a peer of @angular/core@~2.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ng2-table@1.3.2 requires a peer of @angular/forms@~2.0.0 but none is installed. You must install peer dependencies yourself`

**Comment by souchprod** [comment]
Amazing to see this issue still open, please push an updated npm package, that's not a big deal..

**Comment by janurag** [comment]
@nblavoie did you find the workaround because of the dependancy issue i am getting error.

**Comment by nblavoie** [comment]
@janurag which dependency error is blocking your application from compiling/building ? The ones I had on the previous comment ? Those I had were only warnings, but the app is running great even on Angular 5.2.9.

**Comment by ambujkhanna** [comment]
Is this working with Angular 9 or 10 for anyone?

**Comment by breitling** [comment]
I have given up on ng2-table and moved on to NGPrime tables. They are supported and probably way better Angular-wise...good luck!
